tests/staticfiles_tests/test_checks.py

```diff
@@ -132,3 +132,54 @@ def test_nonexistent_directories(self):
             # Nonexistent directories are skipped.
             finder = get_finder("django.contrib.staticfiles.finders.FileSystemFinder")
             self.assertEqual(list(finder.list(None)), [])
+
+
+class StorageCheckTests(SimpleTestCase):
+    DEFAULT_STORAGE_VALUE = {
+        "default": {
+            "BACKEND": "django.core.files.storage.FileSystemStorage",
+        },
+        "staticfiles": {
+            "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
+        },
+    }
+
+    @staticmethod
+    def _remove_storage_alias(alias):
+        """
+        Helper function to remove an alias from the STORAGES setting.
+
+        We need this method because UserSettingsHolder has some backwards compatible
+        code in __setattr__ to keep the aliases 'default' and 'staticfiles' which
+        prevents us from testing what happens when they are missing.
+        """
+        settings.STORAGES.pop(alias, None)
+
+    def test_no_errors_without_setting(self):
+        errors = check_storages(None)
+        self.assertEqual(errors, [])
+
+    @override_settings(STORAGES=DEFAULT_STORAGE_VALUE)
+    def test_no_errors_staticfiles_and_default(self):
+        errors = check_storages(None)
+        self.assertEqual(errors, [])
+
+    @override_settings(STORAGES=DEFAULT_STORAGE_VALUE)
+    def test_no_errors_staticfiles_only(self):
+        self._remove_storage_alias("default")
```

Comment:
```suggestion
        settings.STORAGES.pop("default")
```
ie just put the pop here, 2nd argument unnecessary 🤔